builtin/settingtypes.txt

```diff
@@ -829,10 +829,10 @@ max_block_generate_distance (Max block generate distance) int 6
 map_generation_limit (Map generation limit) int 31000 0 31000
 
 #    Global map generation attributes.
-#    The 'trees' flag only has effect in mgv6.
+#    In Mapgen v6 'decorations' controls non-tree decorations.
```

Comment:
From a translator standpoint : what is "non-tree decorations" exactly ? You have to be the most accurate as possible because some idiomatism in english are hardly translatable in other languages.